tools/ramdump/ramdump_tool.c

```diff
@@ -407,10 +483,51 @@ int main(int argc, char *argv[])
 	}
 
 	printf("Ramdump received successfully..!\n");
+	getchar();
+
+	ret = read(dev_fd, &c, 1);
+	if (ret != 1) {
+		printf("Failed to get synchronization character from device, ret: %d\n", ret);
+		goto init_err;
+	}
+
+	if (c != 'F') {
+		printf("Synchronization character from device was not \'F\', char = %c\n", c);
+		ret = 0;
+		goto init_err;
+	}
+
```

Comment:
same as previous, there is no return.

Reply:
A bit confused.
Does this refer to the comment: -

> There is no return. When we get the RAMDUMP, should we get FSDUMP together always? Is this intended?

Reply:
right. I mean that.

Reply:
Updated user choice method